eng/pipelines/templates/VS-release.template.yml

```diff
@@ -4,6 +4,10 @@ steps:
 
 - template: ../tasks/NuGetToolInstaller.yml
 
+- task: NuGetAuthenticate@0
+  inputs:
+    forceReinstallCredentialProvider: true
```

Comment:
Do we want to keep this?

Reply:
Yep. These are running on Microbuild ScaleSet machines.

Reply:
Right, but our nuget feed is nuget.org, so we don't need authentication

Reply:
Ah, good point.